webapp/portlet/src/main/webapp/vue-apps/common/components/Notifications.vue

```diff
@@ -223,6 +224,11 @@ export default {
     });
   },
   methods: {
+    dispatchDismissed(opened) {
+      if (!opened) {
+        document.dispatchEvent(new CustomEvent('alert-message-dismissed'));
```

Comment:
I propose to add a watch on `snackbar` when it's turned off, in order to centralize the event triggering.

Reply:
already tested that and it didn't work correctly because of delay issues in dispatched event from watch. the issue appears when there several alerts when the second alert forces the close the first one.

Reply:
This may explains more :)
Result when dispatch event in watch sanckbar :
![watch_snackbar](https://github.com/Meeds-io/social/assets/70575401/77c67397-6749-4d75-b46f-907d6838789c)
Result of current work
![usemethod](https://github.com/Meeds-io/social/assets/70575401/4bf2a3ea-7eca-4874-9f03-b6e6143a15ee)

Reply:
ok, seems not clear why it doesn't work, but it didn't as you demonstrate with images. Not blocker for me, even if I would like to analyse more why it didn't work using the watch while it should. Anyway, I resolve the conversation.